doc/Comments.xml

```diff
@@ -343,6 +343,11 @@ use @ExampleSession:
 ]]></Listing>
 It inserts an example into the manual just as @Example would do, but all lines are commented
 and therefore not executed when the file is read.
+All lines have to start with an &AutoDoc; comment (<C>#!</C>).
+The comment will be removed. Also, if the following character is a space,
+it will also be removed. There is never more than one character removed.
```

Comment:
two times "also" in one sentence: not nice.

Also: you remove 2 or 3 characters, so what do you mean with "never more than one character removed" ? Perhaps "space" instead of "character?